LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Crown Copyright (Ministry of Justice)
+Copyright (c) 2022 Crown Copyright (Ministry of Justice)
```

Comment:
Should that read 2020-2022 as per [use-the-correct-copyright-notice](https://technical-guidance.service.justice.gov.uk/documentation/standards/licencing-software-or-code.html#use-the-correct-copyright-notice)